client/src/routes/root.jsx

```diff
@@ -9,10 +9,25 @@ const Root = () => {
   const [played, setPlayed] = useState('');
 
   function getDate() {
+    const norskeMåneder = [
```

Comment:
Selv om siden vises på norsk så er fortsatt kodebasen engelsk, skal vi ha norske eller engelske variable navn?

```suggestion
    const norwegianMonths = [
```